src/haddock/modules/sampling/rigidbody/__init__.py

```diff
@@ -77,6 +77,8 @@ def _run(self):
                 # Create a model for the expected output
                 model = PDBFile(output_pdb_fname, path=self.path)
                 model.topology = [e.topology for e in combination]
+                model.ori_name = '+'.join([m.ori_name for m in combination])
```

Comment:
dont need to create the list, straight: `'+'.join(m.ori_name for m in combination)`

Reply:
why not?

Reply:
:wink: